### PR TITLE
fix: Video Room Need further enhancement

### LINKED ISSUE #6242: Video Room: Need further enhancement
**Is your feature request related to a problem? Please describe.**

When an organizer wants to cancel the edit of the video room and wants to go back to the earlier page, has to click on the back button. There is no cancel button currently.
**Describe the solution you'd like**

When an organizer wants to cancel the edit of the video room, can click the cancel and go back to the previous page.
**Describe alternatives you've considered**

Add a Cancel button next to Save button
**Additional context**

c2

### PR DESCRIPTION
**Pull request by afzal442**
Fixes #6242 

#### Short description of what this resolves:
Added cancel button to get back to previous work

#### Changes proposed in this pull request:

- Added cancel button
Screenshot
c2

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/templates/components/side-bar.hbs [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div class="ui left inverted vertical menu sidebar {{if this.sidebarVisible 'visible'}}">
-    <div class="search-bar d-flex items-center space-between m-2 p-2">
+    <div class="hidden-item search-bar d-flex items-center space-between m-2 p-2">
```
Comment (iamareebjamal) [inline app/templates/components/side-bar.hbs:2]: Why

Comment (afzal442) [reply app/templates/components/side-bar.hbs:2]: by mistake, my GitHub branch didn't show the status before.

Comment (afzal442) [reply app/templates/components/side-bar.hbs]: Nope sir. I created a different branch for that. I switched to new branch I remembered. Thanks.

**Comment by MeghalBisht** [comment]
I think this can be extended to Speaker and  Session Create pages as well.

**Comment by afzal442** [comment]
@iamareebjamal should I make other changes as per suggestion or is this ok for now?

**Comment by iamareebjamal** [comment]
This dos not reset the changes made

**Comment by afzal442** [comment]
Any suggestions Sir!

**Review by sachinchauhan2889 — COMMENTED** [review]
This cancel button directs to previous page only. It must also reset or work. Means it must delete out created record and check other cancel buttons in project.

**Comment by afzal442** [comment]
How to restore previous changes. I couldn't find.

**Comment by iamareebjamal** [comment]
No solution for now

**Pull request merged**